infra/auth/main.tf

```diff
@@ -0,0 +1,106 @@
+locals {
+  common_tags = "${map(
+    "project"   , "${var.DSS_INFRA_TAG_PROJECT}",
+    "env"       , "${var.DSS_DEPLOYMENT_STAGE}",
+    "service"   , "${var.DSS_INFRA_TAG_SERVICE}",
+    "Name"      , "${var.DSS_INFRA_TAG_SERVICE}-auth",
+    "owner"     , "${var.DSS_INFRA_TAG_OWNER}",
+    "managedBy" , "terraform"
+  )}"
+}
+
+resource "auth0_resource_server" "dev_api_resource_server" {
+  name        = "Redwood Dev Resource Server"
+  identifier  = var.OIDC_AUDIENCE
+  signing_alg = "RS256"
+
+  allow_offline_access                            = true
+  token_lifetime                                  = 8600
+  skip_consent_for_verifiable_first_party_clients = true
+}
+
+resource "auth0_client" "dss_auth" {
+  name = "data-store-auth"
+  description = "simple auth system for the data-store; managed by Terraform"
+  app_type = "regular_web"
+  is_first_party = true
+  is_token_endpoint_ip_header_trusted = true
+  token_endpoint_auth_method = "client_secret_post"
+  oidc_conformant = true
+  callbacks = [ "https://example.com/callback", "http://localhost:8080" ]
```

Comment:
Should these `example.com` values be updated?

Reply:
done!